src/main/webapp/WEB-INF/tags/shared/feedbackSubmissionEdit/feedbackSubmissionForm.tag

```diff
@@ -41,13 +41,12 @@
             <c:otherwise>
                 <input type="checkbox" name="sendsubmissionemail">
                 Send me a confirmation email
-                <input type="submit" class="btn btn-primary center-block margin-top-7px"
+                <button type="submit" class="btn btn-primary center-block margin-top-7px"
                        id="response_submit_button" data-toggle="tooltip"
                        data-placement="top" title="<%= Const.Tooltips.FEEDBACK_SESSION_EDIT_SAVE %>"
-                       value="Submit Feedback"
                        <c:if test="${data.preview or (not data.submittable)}">
                            disabled style="background: #66727A;"
-                       </c:if>>
+                       </c:if>>Submit Feedback</button>
```

Comment:
Would be nice to move the addition to a new line instead so it's easier to follow the hierarchy.

```
</c:if>>Submit Feedback</button>
```
to
```
<button type="submit"
        ....
        <c:if test="${data.preview or (not data.submittable)}">
            disabled style="background: #66727A;"
        </c:if>>
    Submit Feedback
</button>
```